benchmarks/profiler_sample_loop.rb

```diff
@@ -28,7 +28,8 @@ def create_profiler
     Datadog.shutdown!
 
     # Call collection directly
-    @stack_collector = Datadog.send(:components).profiler.collectors.first
+    # Note -- this is going to be deprecated/removed soon
+    @stack_collector = Datadog.send(:components).profiler.send(:worker)
     @recorder = @stack_collector.recorder
```

Comment:
```suggestion
    @stack_worker = Datadog.send(:components).profiler.send(:worker)
    @recorder = @stack_worker.recorder
```

Reply:
For the same reason as https://github.com/DataDog/dd-trace-rb/pull/3114/files#r1318276131, I don't think it's worth spending time on code that's about to be deleted ;)